WalletWasabi/BitcoinCore/Monitoring/RpcFeeProvider.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using NBitcoin.RPC;
+using WalletWasabi.Bases;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.BlockchainAnalysis.FeesEstimation
```

Comment:
namespace should change to `WalletWasabi.BitcoinCore.Monitoring`

Reply:
Fixed.